### PR TITLE
add default tag latest for docker build and push

### PR DESCRIPTION
**Pull request by mmpetarpeshev**
Add default tag latest for docker images build on master and tag on pr.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -29,7 +32,11 @@ jobs:
         with:
           images: aeternity/aepp-faucet
           tags: |
+            type=raw,value=latest,enable=${{ endsWith(GitHub.ref, 'master') }}
```
Comment (dincho) [inline .github/workflows/main.yml:35]: if I understand it correctly this pushes latest master build as docker latest tag, while we want to push latest tagged version as docker latest tag

Comment (mmpetarpeshev) [reply .github/workflows/main.yml:35]: okey, so only on tag , without on push in master ?

Comment (dincho) [reply .github/workflows/main.yml:35]: - master branch should go to master docker tag
- vX.Y.Z tag should go to docker vX.Y.Z tag
- latest vX.Y.Z tag should go to docker `latest` tag

Comment (mmpetarpeshev) [reply .github/workflows/main.yml:35]: From the docs , I think we need to remove these lines and will add the latest tag:
https://github.com/aeternity/aepp-faucet-nodejs/pull/8/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R38-R39
Will make the change , but not sure how we can test within the PR

Comment (dincho) [reply .github/workflows/main.yml]: You can test it by tagging your PR commits

Comment (mmpetarpeshev) [reply .github/workflows/main.yml]: The default behaviour of metadata action is like in the table below . Isn't what we want ? I will test one more time all events : 
`

Event | Ref | Docker Tags
-- | -- | --
pull_request | refs/pull/2/merge | pr-2
push | refs/heads/master | master
push | refs/heads/releases/v1 | releases-v1
push tag | refs/tags/v1.2.3 | v1.2.3, latest
push tag | refs/tags/v2.0.8-beta.67 | v2.0.8-beta.67, latest

`

Comment (dincho) [reply .github/workflows/main.yml]: kind of, pr-2 and releases-v1 makes no sense to me, who is gonna clean it up ?

Comment (mmpetarpeshev) [reply .github/workflows/main.yml]: We currently have. only event on push in master and docket tag will be generated only if a tag is pushed/created. 
Lets we clear what we want and I will configure it.

Comment (dincho) [reply .github/workflows/main.yml]: I'd say we want:
Event | Ref | Docker Tags
-- | -- | --
push | refs/heads/master | master
push tag | refs/tags/v1.2.3 | v1.2.3, latest
push tag | refs/tags/v2.0.8-beta.67 | v2.0.8-beta.67

**Comment by marc0olo** [comment]
@dincho @mmpetarpeshev this topic becomes more urgent recently due to the state channel demo we wanna present.

my assumption is that still the old Python faucet is deployed at https://faucet.aepps.com - can you confirm that?

the problem is we will probably have many people simultaneously trying to fund their wallets with the faucet when playing around with that demo. I assume there is an error in the nonce-handling right now.

we should:
- deploy nodejs faucet
- fix nodejs faucet to be able to handle many requests simultaneously and use the right nonce **(if needed)**
    - I know there is a feature in the node now to return the correct nonce based on what tx's are known in the mempool to avoid re-usage and fill gaps. this probably may not work if node is behind loadbalancer, right?! cc @velzevur 

cc: @davidyuk @velzevur @Kalovelo

is it possible we can achieve that in the upcoming weeks before the conference?

**Comment by marc0olo** [comment]
@mmpetarpeshev if we have it deployed in the k8s environment and accessible there we could just point to that URL. then the switch of the main-url wouldn't be that urgent of course.

but for fixing issues in this faucet here we should have auto-deployment enabled in case we need to fix sth.

**Comment by dincho** [comment]
@marc0olo this PR is about docker tags

**Comment by marc0olo** [comment]
sorry, opened an issue

**Pull request closed without merging**